hera_qm/firstcal_metrics.py

```diff
@@ -366,6 +385,25 @@ def __init__(self, calfits_file):
         self.delay_avgs = np.median(self.delays, axis=1)
         self.delay_fluctuations = (self.delays.T - self.delay_avgs).T
 
+        # use gaussian process model to subtract underlying mean function
+        if use_gp is True and self.sklearn_import is True:
+            kernel = gp.kernels.RBF(length_scale=0.2, length_scale_bounds=(0.01, 1.0)) + gp.kernels.WhiteKernel(noise_level=0.01)
```

Comment:
Some of the hard-coded values here are a little opaque. Could you add comments explaining them and/or derive them programmatically?